# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_brotli_http.js
# Commit: dc2ad0369b86
# Full Hash: dc2ad0369b8660e3a5e039e03e224122eb46dcfc
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-09 21:19:46
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 9fb8ccb2f305 -r dc2ad0369b86 netwerk/test/unit/test_brotli_http.js
--- a/netwerk/test/unit/test_brotli_http.js	Tue Apr 09 12:31:55 2024 +0000
+++ b/netwerk/test/unit/test_brotli_http.js	Tue Apr 09 12:31:56 2024 +0000
@@ -100,7 +100,7 @@
     });
     equal(
       Services.prefs.getCharPref("network.http.accept-encoding.secure"),
-      "gzip, deflate, br"
+      "gzip, deflate, br, zstd"
     );
     let { req, buff } = await new Promise(resolve => {
       let chan = NetUtil.newChannel({